source/unified-test-format/unified-test-format.rst

```diff
@@ -462,6 +477,90 @@ The structure of this object is as follows:
 
     Test files SHOULD NOT use this option unless one or more command monitoring
     events are specified in `observeEvents <entity_client_observeEvents_>`_.
+    
+  - ``storeEventsAsEntities``: Optional map of event names to entity names.
+    If provided the test runner MUST:
+    
+      - For each entity name, create the respective entity with a type of
+        "event list". If multiple event names map to the same entity name,
+        exactly one entity MUST be created. If the entity already exists
+        (such as from a previous ``storeEventsAsEntities`` declaration from
+        another client), the test runner MUST raise an error.
+      - Set up an event subscriber for each event named. The event subscriber
+        MUST serialize the events it receives into a document and append
+        the document to the array stored in the specified entity.
+        The fields in the document MUST be as follows:
+        
+        - For ``CommandStartedEvent``:
+       
+          - ``name``: the name of the event, i.e. ``CommandStartedEvent``.
+          - ``commandName``: the name of the command, e.g. ``insert``.
+          - ``startTime``: the (floating-point) number of seconds since
```

Comment:
Later, I suggest making this field consistent with whatever is used in the CMAP events (e.g. `time`, `observedAt`).

If we can offload the started and succeeded/failed correlation to the workload executor, then this can always be documented as the "time that the event was observed". Since the command monitoring spec doesn't actually report this field, the best we can do is calculate it in the listener.